pages/k8s/upgrading.md

```diff
@@ -355,6 +355,16 @@ juju run-action kubernetes-worker/1 upgrade
 ...
 ```
 
+<a id='upgrading-series'> </a>
+
+## Upgrading the Machine's Series
+
+All of the charms support [upgrading the machine's series via Juju](https://juju.is/docs/upgrading-series).
+As each machine is upgraded, the applications on that machine will be stopped and the unit will
+go into a `blocked` status until the upgrade is complete. For the worker units, pods will be drained
+from the node and onto one of the other nodes at the start of the upgrade, and the node will be removed
+from the pool until the upgrade is complete.
```

Comment:
The content here seems fine. I don't think there's much else to say.

The placement here makes the following "Verify your upgrade" section read a bit weird though:

>The output from:
> ```
> juju status
> ```
> ... should now indicate the selected version of Kubernetes is running.

That text made sense when it was immediately after the "Upgrading Kubernetes" section, but not so much now that it's immediately following "Upgrading the Machine's Series".

I recommend either changing "Verify your upgrade" to be a subsection of "Upgrading Kubernetes", or leave it where it is and change the text to something broader than just Kubernetes version, e.g. "... should show all units as active, and should show the correct version of Kubernetes." Or something like that.

Reply:
@johnsca just a quick question - it seems that these changes should apply retrospectively too (as in I should regenerate the versioned upgrade docs also)

Reply:
@evilnick Yes.  I wasn't sure how to do that without manually editing them.  Ideally, I'd love to see it work the other way around: individual release notes pages which are used to generate the current 3 and historical combined / index pages.

Reply:
@johnsca, yes, that's exactly how I'd like to do it, but its a bit chicken and egg. Now the component folders exist, and we will be doing notes/docs starting with beta releases, I think we can do it the other way round.

Reply:
> The placement here makes the following "Verify your upgrade" section read a bit weird: ...

This concern has been resolved. Thanks!

> it seems that these changes should apply retrospectively too (as in I should regenerate the versioned upgrade docs also)

I guess I'm not following here. The charms won't actually support series upgrades until Charmed Kubernetes 1.19. Why would we include series upgrades in docs for older versions?

Reply:
@Cynerva I think the confusion is probably because I put the keepalived clarifications into this PR as well.  Maybe I should split them into two PRs.